okhttp/src/main/java/okhttp3/internal/connection/RealConnection.java

```diff
@@ -104,6 +108,10 @@
 
   /** Nanotime timestamp when {@code allocations.size()} reached zero. */
   public long idleAtNanos = Long.MAX_VALUE;
+  private List<String> subjectAlternativeNames;
+
+  private Map<String, Boolean> supportedHostCache =
```

Comment:
probably wants Javadoc . . . supports _what_ ?
Also final.
Also `Map<String, Boolean>` is kind of a weird type; usually that’s a `Set<String>`...